package.json

```diff
@@ -168,7 +168,7 @@
     "npm": "3.10.10"
   },
   "scripts": {
-    "preanalyze-bundles": "cross-env WEBPACK_OUTPUT_JSON=1 CALYPSO_ENV=production NODE_ARGS=\"--max_old_space_size=8192\" npm run -s build",
+    "preanalyze-bundles": "cross-env WEBPACK_OUTPUT_JSON=1 CALYPSO_ENV=production npm run -s env -- node --max_old_space_size=8192 ./node_modules/webpack/bin/webpack.js --config webpack.config.js --profile --json > stats.json",
```

Comment:
Instead of `./node_modules/webpack/bin/webpack.js` script, you can also run `./node_modules/.bin/webpack`. I'm not sure which one is more "correct".

Reply:
ah, i didn't realize you could run `node ./node_moduels/.bin/webpack`.  I'll update and merge